modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -602,6 +604,25 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
         }
     }
 
+    @Override
+    public void postProcessParameter(CodegenParameter parameter) {
+        super.postProcessParameter(parameter);
+        // in order to avoid name conflicts, we map parameters inside the functions
+        String in_function_identifier = "";
+        if (this.useSingleRequestParameter) {
+            in_function_identifier = "params." + parameter.paramName;
```

Comment:
can we rename `in_function_identifier` to `inFunctionIdentifier` instead to conform to java variable naming convention?